charts/consul/templates/client-daemonset.yaml

```diff
@@ -202,6 +202,13 @@ spec:
             - |
               CONSUL_FULLNAME="{{template "consul.fullname" . }}"
 
+              {{- if (and .Values.dns.enabled .Values.dns.enableRedirection)}}
+              export kube_dns_service_ip=$(cat /etc/resolv.conf | grep nameserver | cut -d' ' -f2)
```

Comment:
should we rename this to be `nameserver_ip`?

Reply:
Also, should we only do that if servers are disabled?

Reply:
The dns-service points to both servers as well as clients (which I was surprised by but the label selector matches on clients and servers) so we might need to add a recursor here as well, unless I'm misunderstanding something

Reply:
Hmm i don't think it does atm. It will select only the pods that have the `hasDNS: "true"` label and clients actually don't have this label right now. I think we'll need to add it if servers are disabled, otherwise it should only point at servers I think.

Reply:
That's what I thought too: https://github.com/hashicorp/consul-k8s/blob/03d4b932256b68ba9fdccba4c00951bb3de275be/charts/consul/templates/client-daemonset.yaml#L35

Reply:
Oh you're right!